roles/docker/defaults/main.yml

```diff
@@ -7,3 +7,6 @@ docker_lvm_data_volume_size: 40%FREE
 docker_lvm_data_volume_size_min: 2G
 
 docker_lvm_auto_extend_pool: yes
+
+docker_cleanup_excludes: []
```

Comment:
It might be clearer if these were named `docker_cleanup_image_excludes`, mirroring your scheme for the `container_excludes`. Not a big deal though, this could definitely be merged as-is.